internal/pkg/cli/validate.go

```diff
@@ -740,25 +739,6 @@ func validatePubSubName(name string) error {
 	return nil
 }
 
-func validateTopicsExist(subscriptions []manifest.TopicSubscription, topicARNs []string, app, env string) error {
```

Comment:
was this unused? 😮

Reply:
Ya it was moved to `cli/deploy` and this one became unused.

Reply:
hmm why isn't it get caught by static check then

Reply:
I think it's because the test remained in `cli` package and the static check counted that as "used"